index.html

```diff
@@ -1243,21 +1243,48 @@ <h2>Evidence</h2>
         <h2>Disputes</h2>
 
         <p>
-The time may come when an <a>entity</a> wants dispute a <a>credential</a>
-issued by another entity. The mechanism for doing this is the same as issuing
-a regular credential except that the <a>subject</a> identifier for the
-<a>claims</a> are those of the disputed credential. For example, if a disputed
-credential with an identifier of <code>https://example.org/credentials/245</code>
-contains disputed statements, an entity would issue the following credential
-in a public venue to make it known that the credential is disputed:
-        </p>
+The time may come when an <a>entity</a> wants to dispute a <a>credential</a>
+issued by the issuer. There at least two different cases to consider:
+a subject disputes a claim made by the issuer, e.g. the address property is out
+of date, or an entity disputes a (false) claim made by the issuer about a different subject, e.g. an imposter is claiming the entity's social security number. 
+Only the subject of a verifiable credential, or its authorised
+agent, is entitled to issue a "DisputeCredential". A "DisputeCredential" issued by anyone other than the subject, or its authorised agent, SHOULD be disregarded by the verifier, unless the verifier has some out of band means of ascertaining the truth of
+the dispute. This is to prevent denial of service attacks whereby an attacker
+falsely disputes a true claim. 
+
+The mechanism for issuing a "DisputeCredential"
+is the same as issuing a regular credential except that the <a>subject</a> identifier
+for the <a>claim</a> in the "DisputeCredential" is the identifier of the disputed credential.  For example, if a credential with an identifier of
+<code>https://example.org/credentials/245</code>
+is disputed, an entity may issue one of the following credentials. In the former case
+the subject might present this to the verifier along with the disputed credential. In
+the later case, the entity might publish the "DisputeCredential"
+in a public venue to make it known that the credential is disputed.
+        </p>
+
+        <pre class="example nohighlight" title="Subject disputes a credential">
+{
+  "id": "http://example.com/credentials/123",
+  "type": ["VerifiableCredential", "DisputeCredential"],
+  <span class="highlight">"claim": {
+    "id": "http://example.com/credentials/245",
+    "currentStatus": "Disputed",
+    "statusReason": "Address is out of date",
+    "address": "10 Some Street, Anytown, ThisLocal, Country X"
+  }</span>,
+  "issuer": "https://example.com/people#me",
+  "issued": "2017-12-05T14:27:42Z",
+  "proof": { ... }
+}
+        </pre>
+
 
-        <pre class="example nohighlight" title="Expressing a disputed credential">
+        <pre class="example nohighlight" title="Another entity disputes a credential">
```

Comment:
I don't understand what is meant by "Another".

Reply:
This is to cover the case where an imposter has managed to get my SS number in his VC. In this case, I am not the subject of the VC, as the imposter is the subject. So I am 'Another entity' who is publishing the dispute in a public place. We cannot replace 'another entity' by 'subject' or 'real subject' because it might be a DOS attack. So in reality, we have no idea who this other entity is, except by some OOB unspecified means.